qiskit/extensions/standard/__init__.py

```diff
@@ -57,3 +58,8 @@
 from .rx import CrxGate
 from .ry import CryGate
 from .rz import CrzGate
+
+warnings.warn('The module qiskit.extensions.standard is deprecated as of 0.14.0 and will be '
+              'removed no earlier than 3 months after the release. You should import the '
+              'standard gates from qiskit.circuit.library.standard_gates instead.',
+              DeprecationWarning, stacklevel=2)
```

Comment:
@ajavadia Importing from `qiskit.extensions.standard` should throw an error. However from `qiskit.extensions` we couldn't find a solution to deprecate that, without also deprecating the other things in there (simulator and quantum_info) and decided to allow that still since there's no overhead. Is that ok?

Reply:
ok i see, i was just hoping that we can start the deprecation process on `qiskit.extensions`. Because if we don't then we can't really remove the gates from there in 3 months since people may be importing from `qiskit.extensions`. 

But let's merge this for now and if we have time to move the other stuff out of qiskit.extensions into the circuit library then we can deprecate the whole namespace.